configuration/debian/c8y-firmware-plugin/postinst

```diff
@@ -1,8 +1,3 @@
 #!/bin/sh
 
-set -e
```

Comment:
We shouldn't remove `set -e` here as the script should stop on unexpected errors (e.g. during the `#DEBHELPER#` section that is injected by cargo-deb)

Reply:
If all that this script is doing is running `#DEBHELPER#`, is it even needed? Or a `postint` script will be  auto-generated in the absence of a custom one?

Reply:
Good question. From memory it is auto generated if the postinst file does not exist, so maybe that is the better option. You can verify the behaviour by just inspecting the deb file afterwards